pkg/provider/provider.go

```diff
@@ -166,7 +167,10 @@ func (p *KedaProvider) GetExternalMetric(ctx context.Context, namespace string,
 			}
 			// Filter only the desired metric
 			if strings.EqualFold(metricSpec.External.Metric.Name, info.Metric) {
+				startTime := time.Now()
```

Comment:
We are probably going to remove this part of the code soon, so I am not sure, whether it make sense to add a new functionality here. @JorTurFer WDYT?

Reply:
I wouldn't add anything there because it's deprecated, the metric should be just in the operator IMO

Reply:
Sure, removed as well.